### DISCUSSION TITLE
Incrementing Version number

### DISCUSSION THREAD

**Discussion by rtfmoz2** [ArturoDent/find-and-transform#9] [General]
Want to implement a version string that increments on every file open. Was hoping I could use this tool to implement.

\#\#\#\# Save Version 1234556 \#\#\#\#

**Comment by ArturoDent** [comment]
It would be easy to write a keybinding that would search for `#### Save Version (\\d+) ####` and replace it with an increment of your choice.  But you would have to manually trigger it.  This extension cannot run automatically upon file open.  

However, you may be able to put the same find/replace into a setting that is run on every save.  Is that an acceptable alternative?
